gradle-plugin/sample/src/main/java/io/realm/example/sample/models/Person.java

```diff
@@ -16,10 +16,12 @@
 package io.realm.example.sample.models;
 
 import io.realm.RealmObject;
+import io.realm.annotations.Ignore;
 
 public class Person extends RealmObject {
```

Comment:
That will require changes to the annotation processor, which is not part of this PR.
The changes to the annotation processor will have a huge impact on the realm unit tests